### PR TITLE
[FIX] l10n_syscohada: Fix COA

### PR DESCRIPTION
**Pull request by malb-odoo**
The account 81 to 89 were misclassified, this PR fix the issue

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_syscohada/data/account.account.template.csv [hunk 1]
```diff
@@ -1014,51 +1014,51 @@
...
+"pcg_849","REPRISES DE CHARGES POUR DEPRECIATIONS ET PROVISIONS POUR RISQUES A COURT TERME H.A.O.","849","account.data_account_type_revenue","l10n_syscohada.syscohada_chart_template","False"
+"pcg_851","DOTATIONS AUX PROVISIONS RÉGLEMENTÉES","851","account.data_account_type_expenses","l10n_syscohada.syscohada_chart_template","False"
+"pcg_852","DOTATIONS AUX AMORTISSEMENTS H.A.O.","852","account.data_account_type_expenses","l10n_syscohada.syscohada_chart_template","False"
+"pcg_853","DOTATIONS AUX DÉPRÉCIATION H.A.O.","853","account.data_account_type_expenses","l10n_syscohada.syscohada_chart_template","False"
+"pcg_854","DOTATIONS AUX PROVISIONS POUR RISQUES ET CHARGES H.A.O.","854","account.data_account_type_expenses","l10n_syscohada.syscohada_chart_template","False"
+"pcg_858","AUTRES DOTATIONS H.A.O.","858","account.data_account_type_expenses","l10n_syscohada.syscohada_chart_template","False"
+"pcg_861","REPRISES DE PROVISIONS REGLEMENTEESREPRISES DE PROVISIONS RÉGLEMENTÉES","861","account.data_account_type_revenue","l10n_syscohada.syscohada_chart_template","False"
```
Comment (jco-odoo) [inline addons/l10n_syscohada/data/account.account.template.csv:1043]: Something is off here with the name

File: addons/l10n_syscohada/data/account.account.template.csv [hunk 2]
```diff
@@ -1130,4 +1130,4 @@
 "pcg_96","COMPTES D'ECARTS SUR COUTS PREETABLIS","96","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
 "pcg_97","COMPTES DE DIFFERENCES DE TRAITEMENT COMPTABLE","97","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
 "pcg_98","COMPTES DE RESULTATS","98","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
-"pcg_99","COMPTES DE LIAISONS INTERNES","99","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
\ No newline at end of file
+"pcg_99","COMPTES DE LIAISONS INTERNES","99","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
```
Comment (jco-odoo) [inline addons/l10n_syscohada/data/account.account.template.csv:1133]: unnecessary diff?

**Review by jco-odoo — COMMENTED** [review]
@malb-odoo I hope you do not mind quickly checking this still.

File: addons/l10n_syscohada/data/account.account.template.csv (hunk 1 above)
Comment (malb-odoo) [reply addons/l10n_syscohada/data/account.account.template.csv:1043]: I've check in the official COA, it seems good, what do you find off ?

File: addons/l10n_syscohada/data/account.account.template.csv (hunk 2 above)
Comment (malb-odoo) [reply addons/l10n_syscohada/data/account.account.template.csv:1133]: There was a missing empty line at the end of the file, i think git recognize that as a diff

File: addons/l10n_syscohada/data/account.account.template.csv (hunk 1 above)
Comment (jco-odoo) [reply addons/l10n_syscohada/data/account.account.template.csv:1043]: @malb-odoo Is it normal that the name is double?

Comment (malb-odoo) [reply addons/l10n_syscohada/data/account.account.template.csv:1043]: Ah didn't saw the last one ahah, yep will changed that

**Comment by jco-odoo** [comment]
It might be counterintuitive, but apparently this is the P&L part of these things.

**Comment by jco-odoo** [comment]
@robodoo r+
